lib/puppetserver/utils/http_utilities.rb

```diff
@@ -4,36 +4,39 @@
 module Puppetserver
   module Utils
     # Utilities for doing HTTPS against the CA that wraps Net::HTTP constructs
-    module HttpUtilities
+    class HttpUtilities
```

Comment:
I think we should give this a different name if it's going to be an actual class that gets instantiated. Something like `HttpClient`?